Makefile

```diff
@@ -262,3 +262,44 @@ ci: clean static-checks test
 ###############################################################################
 update-pins: update-api-pin
 
+###############################################################################
+# Release
+###############################################################################
+
+## Tags and builds a release from start to finish.
+release: release-prereqs
+	$(MAKE) VERSION=$(VERSION) release-tag
+
+	@echo ""
+	@echo "Release tagged. Next, push the tag."
+	@echo ""
+	@echo "  make VERSION=$(VERSION) release-publish"
+	@echo ""
+
+## Produces a git tag for the release.
+release-tag: release-prereqs release-notes
+	git tag $(VERSION) -F release-notes-$(VERSION)
+
+## Generates release notes.
+release-notes: release-prereqs
+	echo "This release is for build purposes only and this should not be referenced externally." > release-notes-$(VERSION)
+	echo "libcalico is not a stable interface." >> release-notes-$(VERSION)
```

Comment:
Probably worth including a link to `projectcalico/api` since that is the actual public API repo.